doc/man7/EVP_KDF_SS.pod

```diff
@@ -0,0 +1,224 @@
+=pod
+
+=head1 NAME
+
+EVP_KDF_SS - The Single Step EVP_KDF implementation. (Also known as One
+-Step Key Derivation).
+
+=head1 DESCRIPTION
+
+Support for computing the B<SSKDF> KDF through the B<EVP_KDF> API.
```

Comment:
A Key Derivation Function Key Derivation Function?  I would suggest replacing `B<SSKDF> KDF` with `Single Step key derivation function (also known as One-Step key derivation function)`

Reply:
... or actually, now that I've reading the rest, I'd say that you can lose this line entirely.